src/unicodeMake.js

```diff
@@ -47,10 +52,14 @@ for (const letter of letters) {
             const combined2 = combined + accent2;
             const normalized2 = combined2.normalize('NFC');
             if (normalized2.length === 1) {
-                console.log(
-                    `    ${encode(normalized2)}: ${encode(combined2)},`
-                    + `  // ${normalized2} = ${accents[accent].text}`
-                    + `${accents[accent2].text}{${letter}}`);
+                const key = encode(normalized2);
+                if (!keys.has(key)) {
+                    keys.add(key);
+                    console.log(
+                        `    ${encode(normalized2)}: ${encode(combined2)},`
+                        + `  // ${normalized2} = ${accents[accent].text}`
+                        + `${accents[accent2].text}{${letter}}`);
```

Comment:
The reason why there are duplicates is b/c `\c\'{c}` and `\'\c{c}` map to the same character.  Same goes for the uppercase version.

Reply:
Thanks for fixing this!

I checked that there are still no 3-accent combos that resolve to a single character.

```js
            for (const accent3 of Object.getOwnPropertyNames(accents)) {
                if (accent === accent3 || accent2 === accent3) {
                    continue;
                }
                const combined3 = combined2 + accent3;
                const normalized3 = combined2.normalize('NFC');
                if (normalized3.length === 1) {
                    const key = encode(normalized3);
                    if (!keys.has(key)) {
                        keys.add(key);
                        console.log(
                            `    ${encode(normalized3)}: ${encode(combined3)},`
                            + `  // ${normalized3} = ${accents[accent].text}`
                            + `${accents[accent2].text}{${letter}}`
                            + `${accents[accent3].text}{${letter}}`);
                    }
                }
            }
```